tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/manifest.py

```diff
@@ -62,8 +62,9 @@ def load_manifest_from_CxG() -> List[Dataset]:
         schema_version = dataset["schema_version"]
 
         if schema_version not in CXG_SCHEMA_VERSION_IMPORT:
-            logging.warning(f"Dropping dataset {dataset_id} due to unsupported schema version")
-            continue
+            msg = f"Dropping dataset {dataset_id} due to unsupported schema version"
```

Comment:
I think the message should reflect the new behavior. Something like `Aborting build due to {dataset_id} having unsupported version {schema_version}`.

Reply:
doh!   Will fix.  TY

Reply:
this also reminds me that we need to check for this in the H5AD itself.  Will add that as well.